### PR TITLE
[DEV-11526] Feature - Auto-convert pasted URLs to embeds

### PR DESCRIPTION
**Pull request by e1himself**
[Screencast from 2023-09-07 15-07-32.webm](https://github.com/prezly/slate/assets/370680/9138abe2-9e33-47d4-813c-ca1991ec156e)

### CODE REVIEW

File: packages/slate-editor/src/extensions/placeholders/PlaceholdersExtension.tsx [hunk 1]
```diff
@@ -96,6 +97,7 @@ export interface Parameters {
     withSocialPostPlaceholders?: false | { fetchOembed: FetchOEmbedFn };
     withVideoPlaceholders?: false | { fetchOembed: FetchOEmbedFn };
     withWebBookmarkPlaceholders?: false | { fetchOembed: FetchOEmbedFn };
+    withUnfurlingPastedUrls?: false | { fetchOembed: FetchOEmbedFn };
```
Comment (oleg-semyonov) [inline packages/slate-editor/src/extensions/placeholders/PlaceholdersExtension.tsx:100]: Just nitpicking here, but I believe that `withPastedUrlsUnfolding` sounds a bit more correct 🤔

**Review by oleg-semyonov — APPROVED** [review]

File: packages/slate-editor/src/extensions/placeholders/PlaceholdersExtension.tsx (hunk 1 above)
Comment (e1himself) [reply packages/slate-editor/src/extensions/placeholders/PlaceholdersExtension.tsx:100]: I agree to the order. Will change! :ok_hand: 

But do you also insist on changing _unfurling_ to _unfolding_?

Comment (e1himself) [reply packages/slate-editor/src/extensions/placeholders/PlaceholdersExtension.tsx:100]: I'm leaving it as _unfurling_, as this seems to be a common industry term for this functionality

image

Comment (oleg-semyonov) [reply packages/slate-editor/src/extensions/placeholders/PlaceholdersExtension.tsx]: > I'm leaving it as unfurling, as this seems to be a common industry term for this functionality

Makes perfect sense!

**Pull request merged**